core/shared/src/main/scala/canoe/api/models/CallbackQueryAPI.scala

```diff
@@ -23,6 +23,12 @@ final class CallbackQueryAPI(private val query: CallbackQuery) extends AnyVal {
   def alert[F[_]: TelegramClient](text: String): F[Boolean] =
     AnswerCallbackQuery.alert(query.id, text).call
 
+  /**
+    * React with no notification
+    */
+  def withoutNotification[F[_]: TelegramClient]: F[Boolean] =
```

Comment:
Let's rename `withoutNotification` to `finish`. It follows the pattern of using verbs to denote actions like `query.alert`, `query.inform`, `query.redirect`. Subjectively, it gives a better intuition of the underlying action.

Please compare:
```
for {
  _ <- query.message.traverse(_.chat.send(cbd))
  _ <- query.finish
}
```
vs.
```
for {
  _ <- query.message.traverse(_.chat.send(cbd))
  _ <- query.withoutNotification
}
```

Reply:
@augustjune, agree. looks good.